test/fixtures/StTable.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
Rewriting the inline table component into a Vue SFC was not required but Vue 3 has breaking changes for render functions, not Vue SFC template.